pkg/app/app.go

```diff
@@ -390,26 +390,29 @@ func initClients(c *config.Config) error {
 	// method on the app. Forces developers at authorship time to think about
 	// dependencies / make sure things are ready.
 	log.Notice("Initializing clients...")
-	log.Debug("Trying to connect to etcd")
 
-	// Intialize the etcd configuration
-	clients.InitEtcdConfig(c)
-	etcdClient, err := clients.Etcd()
-	if err != nil {
-		return err
-	}
+	if strings.ToLower(c.GetString("dao.type")) != "crd" {
```

Comment:
What happens if it isn't crd? What is the other valid value?

Reply:
All other values will mean we use `etcd`. This is so every old config should still work as expected with `etcd` as it's DAO.